# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1202469765 -3600
#      Fri Feb 08 12:22:45 2008 +0100
# Node ID feaaf725c54f26da380e168ef7bfd9f89ff65f52
# Parent  edbaa13397ee708ebbadfa794cf69c6403f53e33
Ignore build directories.
* * *
Ignore configure-related files.

diff --git a/.hgignore b/.hgignore
--- a/.hgignore
+++ b/.hgignore
@@ -1,3 +1,8 @@
 (^|/)CVS($|/)
 (^|/)\.hg($|/)
 (^|/)\.hgtags($|/)
+
+^build-.*($|/)
+(^|/)configure$
+(^|/)autom4te.cache($|/)
+^config\.h\.in$
diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2008-06-04  Michael Goffioul <michael.goffioul@gmail.com>
+
+	* .hgignore: Also ignore build-.*, configure, and autom4te.cache.
+
 2008-05-22  Jaroslav Hajek <highegg@gmail.com>
 
 	* mk-opts.pl (emit_print_function, emit_set_function,
 	emit_show_function): Support float type.
 
 2008-05-20  David Bateman  <dbateman@free.fr>
 
 	* configure.in (AC_CHECK_FUNCS): Add expm1, lgammaf, lgammaf_r,
